src/main/webapp/WEB-INF/tags/base.tag

```diff
@@ -0,0 +1,43 @@
+<%@tag description="Main base template for chat app"%>
```

Comment:
This is a base template that will reduce a lot of the redundancy of the having to rewrite the header of the chat app for every JSP view. It also keeps maintainability easy; want to add a tab to the app? Just add it here.